src/DraxList.tsx

```diff
@@ -652,7 +659,10 @@ export const DraxList = <T extends unknown>(
 					{...props}
 					ref={setFlatListRefs}
 					renderItem={renderItem}
-					onScroll={onScroll}
+					onScroll={(scrollEvent) => {
+						onScroll(scrollEvent);
+						if (onScrollProp !== undefined) onScrollProp(scrollEvent);
```

Comment:
This logic change should be moved into the `onScroll` `useCallback` definition rather than an anonymous function here.